Doc/library/dis.rst

```diff
@@ -924,7 +924,7 @@ iterations of the loop.
    Unpacks ``STACK[-1]`` into *count* individual values, which are put onto the stack
    right-to-left::
 
-      STACK.extend(STACK.pop()[:count:-1])
+      STACK.extend(STACK.pop()[:-count-1:-1])
```

Comment:
Shouldn't we also document somehow that we require `STACK.pop()` to have exactly `count` elements?

Reply:
Update the sentence "Unpacks ``STACK[-1]`` into *count* individual values" into "Unpacks ``STACK.pop()`` into *count* individual values" will be enough?

Reply:
Done!